Add details for AEP based on upcoming suite release (#652)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/configuration/commerce-configuration.mdx

**Configuration Properties:**

> - **`aep-ims-org-id`** - (Optional) Adobe IMS Organization ID for Experience Platform integration (e.g., `"1234567890ABCDEF7F000101@AdobeOrg"`)

Reviewer: ```suggestion
- **`aep-ims-org-id`** - (Optional) Adobe IMS Organization ID for Experience Platform integration (for example, `"1234567890ABCDEF7F000101@AdobeOrg"`)
```

> - **`aep-datastream-id`** - (Optional) Datastream ID for routing data to Adobe Experience Platform (e.g., `"12345678-1234-1234-1234-123456789012"`)

Reviewer: ```suggestion
- **`aep-datastream-id`** - (Optional) Datastream ID for routing data to Adobe Experience Platform (for example, `"12345678-1234-1234-1234-123456789012"`)
```

> - **`base-currency-code`** - The base currency code for the store (e.g., "USD", "EUR")

Reviewer: ```suggestion
- **`base-currency-code`** - The base currency code for the store (for example, "USD", "EUR")
```

**Configuration Properties:**

> - **`aep-ims-org-id`** - (Optional) Adobe IMS Organization ID for Experience Platform integration (e.g., `"1234567890ABCDEF7F000101@AdobeOrg"`)

Reviewer: ```suggestion
- **`aep-ims-org-id`** - (Optional) Adobe IMS Organization ID for Experience Platform integration (for example, `"1234567890ABCDEF7F000101@AdobeOrg"`)
```

> - **`aep-datastream-id`** - (Optional) Datastream ID for routing data to Adobe Experience Platform (e.g., `"12345678-1234-1234-1234-123456789012"`)

Reviewer: ```suggestion
- **`aep-datastream-id`** - (Optional) Datastream ID for routing data to Adobe Experience Platform (for example, `"12345678-1234-1234-1234-123456789012"`)
```

> - **`base-currency-code`** - The base currency code for the store (e.g., "USD", "EUR")

Reviewer: ```suggestion
- **`base-currency-code`** - The base currency code for the store (for example, "USD", "EUR")
```
